.travis.test

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+set -ex
+
+# enable coprs
+dnf install -y dnf-plugins-core
+dnf copr enable -y npmccallum/jansson
+# dnf copr enable -y npmccallum/jose
+
+# install build dependencies
+dnf install -y \
+    gcc redhat-rpm-config make pkgconfig \
+    findutils tar bzip2 \
+    jansson-devel openssl-devel zlib-devel \
+    python-tox python-pip Cython
+
+# dnf install -y jose libjose-devel libjose-openssl-devel libjose-zlib-devel
+
+# install Python dependencies
+case "$1" in
+  python2)
+    dnf install -y python-devel python-setuptools python-virtualenv
+    ;;
+  python3)
+    dnf install -y python3-Cython python3-devel python3-setuptools python3-tox python3-virtualenv
+    ;;
+  *)
+    echo "$0 python2|python3"
+    exit 1
+    ;;
+esac
+
+# install jose
```

Comment:
It still isn't in anything but rawhide. Jose is just developing too fast. I'm pushing builds regularly to copr until the pace slows down. So I suggest using that repo.